packages/common/src/index.ts

```diff
@@ -508,11 +509,20 @@ export type DbtColumnLightdashMetric = {
     sql?: string;
     hidden?: boolean;
     round?: number;
+    format?: string;
 };
 
+export type ResultRow = {
+    [col: string]: {
+        [value: string]: {
```

Comment:
Shouldn't this be
```suggestion
        value: {
```

Otherwise we are saying the column object can have unlimited string properties. e.g
```
{
  'my_table_my_dimension': {
      'test': {
           raw: '1',
           formated: '1' 
       }
        'test22': {
           raw: '1',
           formated: '1' 
       }
  }
}
```